kotlin-eclipse-core/src/org/jetbrains/kotlin/core/resolve/KotlinCoroutinesScope.kt

```diff
@@ -0,0 +1,24 @@
+package org.jetbrains.kotlin.core.resolve
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.joinAll
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import kotlin.coroutines.CoroutineContext
+
+object KotlinCoroutinesScope : CoroutineScope {
```

Comment:
I really don't like name of this object.
It can by `KotlinAnalyzisScope` or `KotlinSupervisedScope` if we want to use it somewhere else.